src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -47,6 +67,63 @@ def create_results_link(output_dir: Path, link_name: Path) -> None:
     link_dir.symlink_to(output_dir, target_is_directory=True)
 
 
-def perform_post_processing(raw_output_dir: Path, final_output_dir: Path) -> None:
-    shutil.copytree(raw_output_dir, final_output_dir)
-    generate_names(raw_output_dir, final_output_dir)
+def perform_post_processing(
+    raw_output_dir: Path, final_output_dir: Path, artifact_path: Path
+) -> None:
+    raw_results = load_data(raw_output_dir)
+    # Generate all post-processing maps to apply to raw results
+    artifact = Artifact(artifact_path)
+    maps = generate_maps(raw_results, artifact)
+
+    # Iterate through expected forms and generate them. Generate columns each of these forms need to have.
+    for observer in FINAL_OBSERVERS:
+        # Fixme: This code assumes a 1 to 1 relationship of raw to final observers
+        obs_data = raw_results[observer]
+
+        obs_data = obs_data.drop(columns=COLUMNS_TO_NOT_MAP[observer])
```

Comment:
As currently written, this will return a dataframe that has both a `first_name` column and a `first_name_id` column. We want to drop the `first_name_id` column.